packages/api-server/api_server/routes/fleets.py

```diff
@@ -63,3 +74,108 @@ async def sub_fleet_log(_req: SubscriptionRequest, name: str):
     return fleet_events.fleet_logs.pipe(
         rxops.filter(lambda x: cast(FleetLog, x).name == name)
     )
+
+
+@router.post("/{name}/decommission", response_model=RobotCommissionResponse)
+async def decommission_robot(
+    name: str,
+    robot_name: str,
+    reassign_tasks: bool,
+    allow_idle_behavior: bool,
+    repo: FleetRepository = Depends(fleet_repo_dep),
+    user: User = Depends(user_dep),
+):
+    """
+    Decommissions a robot, cancels all direct tasks, and preventing it from
+    accepting any new tasks (both dispatch tasks and direct tasks), with the
+    options to:
+
+    - Reassign all queued dispatch tasks to other robots. If task reassignment
+      is chosen, the response will contain the results of the reassignment as
+      well, any failed reassignments will be cancelled instead.
+    - Disable idle behaviors (formerly known as finishing tasks). If disabled,
```

Comment:
The param is `allow_idle_behavior` so the docs should explain it in the context of "allow".

Reply:
https://github.com/open-rmf/rmf-web/pull/920/commits/034400e8d890528c5218903bac15e65f4ac442aa thanks that makes sense